packages/prisma/schema.prisma

```diff
@@ -614,6 +617,7 @@ model Schedule {
   name         String
   timeZone     String?
   availability Availability[]
+  Host         Host[]
```

Comment:
```suggestion
  host         Host[]
```

Reply:
Thankyou! @CarinaWolli for the review, i have updated suggested changes in single commit